ipa-core/src/protocol/ipa_prf/prf_sharding/mod.rs

```diff
@@ -458,17 +458,13 @@ where
 /// # Panics
 /// Propagates errors from multiplications
 #[tracing::instrument(name = "attribute_cap_aggregate", skip_all)]
-pub async fn attribute_cap_aggregate<C, BK, TV, TS, SS, S, F>(
-    sh_ctx: C,
+pub async fn attribute_cap_aggregate<'a, BK, TV, TS, SS, F>(
+    sh_ctx: SemiHonestContext<'a>,
```

Comment:
I remember some conversations around using the actual type rather than trait bound for contexts, but I can't find it for some reason. Is the idea that once we get to malicious implementation we would only support active security? Are we losing the ability of executing both semi-honest and malicious version with the same protocol code by doing this change or not?

Reply:
No, this is just a temporary fix